File: src/main/java/com/elf/vipForElf/domain/test/vo/FloorListVO.java
package com.elf.vipForElf.domain.test.vo;

import com.elf.vipForElf.domain.test.entity.FloorEntity;
import lombok.Value;

@Value
public class FloorListVO {
    Long id;
    Integer floorNumber;

    public FloorListVO(FloorEntity floorEntity){
        this.id = floorEntity.getId();
        this.floorNumber = floorEntity.getFloorNumber();
    }
}

File: src/main/java/com/elf/vipForElf/domain/floor/vo/FloorListVO.kt
package com.elf.vipForElf.domain.floor.vo

import com.elf.vipForElf.domain.floor.entity.FloorEntity

data class FloorListVO(
    val id: Long?,
    val floorNumber: Int
) {
    constructor(floorEntity: FloorEntity) : this(
        id = floorEntity.id,
        floorNumber = floorEntity.floorNumber
    )
}
